### PR TITLE
Replace toStartOfFiveMinute to toStartOfFiveMinutes

### LINKED ISSUE #36512: toStartOfFiveMinute - Minor inconsistency in naming
I guess, function 'toStartOfFiveMinute' should be replaced by 'toStartOfFiveMinutes' (plural s is missing)

### PR DESCRIPTION
**Pull request by Joeywzr**
### Changelog category (leave one):

- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
replaced toStartOfFiveMinute to toStartOfFiveMinutes

Closes #36512

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

File: src/Functions/toStartOfFiveMinute.cpp [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
 {
 
-using FunctionToStartOfFiveMinute = FunctionDateOrDateTimeToSomething<DataTypeDateTime, ToStartOfFiveMinuteImpl>;
+using FunctionToStartOfFiveMinutes = FunctionDateOrDateTimeToSomething<DataTypeDateTime, ToStartOfFiveMinutesImpl>;
 
-void registerFunctionToStartOfFiveMinute(FunctionFactory & factory)
+void registerFunctionToStartOfFiveMinutes(FunctionFactory & factory)
 {
-    factory.registerFunction<FunctionToStartOfFiveMinute>();
+    factory.registerFunction<FunctionToStartOfFiveMinutes>();
```
Comment (amosbird) [inline src/Functions/toStartOfFiveMinute.cpp:13]: Let's add a new alias to avoid breaking backward compatibility.

Comment (Joeywzr) [reply src/Functions/toStartOfFiveMinute.cpp:13]: 👌

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
@mergifyio update

**Comment by alexey-milovidov** [comment]
@mergifyio update

**Review by rfraposa — COMMENTED** [review]
I deleted the unneeded /docs/ja files from, but otherwise the /docs files look good

**Comment by alexey-milovidov** [comment]
@mergifyio update after #36607.

**Comment by alexey-milovidov** [comment]
> Integration tests (asan, actions) [2/3] — fail: 1, passed: 617, flaky: 2 

Bad test, time dependent.

> Stress test (address, actions) — Backward compatibility check: Fatal message in clickhouse-server.log (see 

Fixed in master.

> Style Check (actions) — Check black failed. 

Fixed in master.

> Stateless tests flaky check (address, actions) — Timeout, fail: 0, passed: 466 

As expected.

**Pull request merged**